app/models/search.rb

```diff
@@ -17,9 +17,13 @@ def single_search(params, model)
     if params[:name]
       name = params[:name].gsub("-", " ").downcase
       model.find_by("lower(name) LIKE ?", name)
+    elsif params[:first_name]
+      model.find_by("lower(first_name) LIKE ?", params[:first_name])
+    elsif params[:last_name]
+      model.find_by("lower(last_name) LIKE ?", params[:last_name])
     elsif params[:description]
       description = params[:description].gsub("-", " ").downcase
-      model.find_by("lower(description) LIKE ?", description)
+      model.find_by("lower(description) LIKE ?", "%#{description}%")
```

Comment:
Can we remove the modulos on here? Interpolation leaves us open to SQL injection